pkg/config/deployment.go

```diff
@@ -220,9 +223,13 @@ type WaitStageOptions struct {
 
 // WaitStageOptions contains all configurable values for a WAIT_APPROVAL stage.
 type WaitApprovalStageOptions struct {
+	// number of seconds to timeout wait approval
+	Timeout   int64    `json:"timeout"`
```

Comment:
Could you please change this to use `Duration` instead of int64?
https://github.com/pipe-cd/pipe/blob/master/pkg/config/duration.go#L23

Reply:
OK 👍